app/views/add-payment-hold.njk

```diff
@@ -23,47 +23,50 @@
         <h1 class="govuk-heading-l">Add payment hold</h1>
 
         {% if errors %}
-        <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
-          <h2 class="govuk-error-summary__title" id="error-summary-title">
+          <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+            <h2 class="govuk-error-summary__title" id="error-summary-title">
             There is a problem
           </h2>
-          <div class="govuk-error-summary__body">
-            <ul class="govuk-list govuk-error-summary__list">
-              {% for errorDetail in errors.details %}
-                <span class="govuk-error-message">
-                  <span class="govuk-visually-hidden">Error:</span> {{ errorDetail.message }}
-                </span>
-              {% endfor %}
-            </ul>
+            <div class="govuk-error-summary__body">
+              <ul class="govuk-list govuk-error-summary__list">
+                {% for errorDetail in errors.details %}
+                  <span class="govuk-error-message">
+                    <span class="govuk-visually-hidden">Error:</span>
+                    {{ errorDetail.message }}
```

Comment:
The error which is returned for hold category missing is `"holdCategoryId" is required` would this be better as `"Hold category" is required`?

Reply:
Updated as above